src/PileupTrack.js

```diff
@@ -910,12 +1070,15 @@ PileupTrack.config = {
       '#DCDCDC',
     ],
     outlineReadOnHover: false,
+    outlineMateOnHover: false,
```

Comment:
Could you add this option to the `optionsInfo` list below?

You could also add `minMappingQuality` and `largeInsertSizeThreshold` with a few useful values. `highlightReadsBy` would be a little more difficult since it can take  combinations of values but I suppose you could add some of the single values as options.

Reply:
Hm... turns out that this is actually problematic. These options cannot be switched on currently from the frontend, just via the viewconf. The problem is that the datafetcher needs to be reinitialized, since the data needs to be loaded differently. This happens in the constructor, which can't be invoked from the, e.g., `rerender` function. Is there a way to reload the whole thing without refreshing the page?

Reply:
Yikes, that's a problem I haven't encountered before. Is it possible to recreate the datafetcher in the `reload` method?

Reply:
It turns out I can just overwrite the datafetcher in the rerender method. I also factored out some initialization code to reset everything when options change. That seems to work well. Looking at the `reload` method was a good tip. Thanks!
I added the new options to the options menu, as requested.

Reply:
Awesome, thank you!